web/tests/functional/config/__init__.py

```diff
@@ -0,0 +1,81 @@
+# coding=utf-8
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+
+""" Setup for the config test for the web commands. """
+
+
+import os
+import shutil
+import sys
+
+from libtest import codechecker
+from libtest import env
+from libtest import project
+
+
+# Test workspace should be initialized in this module.
+TEST_WORKSPACE = None
+
+
+def setup_package():
+    """Setup the environment for the tests."""
+
+    global TEST_WORKSPACE
+    TEST_WORKSPACE = env.get_workspace('config')
+
+    # Set the TEST_WORKSPACE used by the tests.
+    os.environ['TEST_WORKSPACE'] = TEST_WORKSPACE
+
+    test_project = 'cpp'
+    test_project_path = project.path(test_project)
+    project_info = project.get_info(test_project)
+
+    # Create a basic CodeChecker config for the tests, this should
+    # be imported by the tests and they should only depend on these
+    # configuration options.
+    codechecker_cfg = {
+        'workspace': TEST_WORKSPACE,
+        'check_env': env.test_env(TEST_WORKSPACE),
+        'reportdir': os.path.join(TEST_WORKSPACE, 'reports'),
+        'checkers': ['-d', 'core.CallAndMessage',
+                     '-e', 'core.StackAddressEscape']
+    }
+
+    # Start or connect to the running CodeChecker server and get connection
+    # details.
+    print("This test uses a CodeChecker server... connecting...")
+    server_access = codechecker.start_or_get_server()
+    server_access['viewer_product'] = 'config'
+    codechecker.add_test_package_product(server_access, TEST_WORKSPACE)
+
+    # Extend the checker configuration with the server access.
+    codechecker_cfg.update(server_access)
+
+    ret = codechecker.analyze(codechecker_cfg, test_project_path)
```

Comment:
Could we limit this test only to store and not analyze, we can save a lot of time during the tests if we do not have to run the analysis.